doc/ruleSets/PHP54Migration.rst

```diff
@@ -0,0 +1,12 @@
+============================
+Rule set ``@PHP54Migration``
+============================
+
+Rules to improve code for PHP 5.6 compatibility.
```

Comment:
PHP54Migration but then you say PHP 5.6, seems weird.

Reply:
typo, already fixed in other PR. thanks